### ISSUE TITLE
No Platform Images (dev branch)

### ISSUE THREAD

**Issue by crunchprank** [Clidus/gwl#65]
When adding a game to your collection, on the "Platforms" section the user's profile shows the GWL logo (/images/platforms/gwl_default.png) for all platforms instead of the images of the platforms.

http://i.imgur.com/l2Qn78s.png

**Comment by Clidus** [comment]
The platforms table is populated as the site is used. So the first time you add a PS4 game, it adds PS4 to the database. Giant Bomb unfortunately don't have a good collection of platform logos, so I had to collect these manually.

Soooooo, when a new platform is added, it has a default image (the site logo) until the image can be added. These images are in /images/platforms/ but they need to be manually linked to a platform in the database.

I've updated database.txt so you don't have to do this manually, but you're going to have to start with a clean database to use this, as you will already have platforms added with different ID's.
